combat-trainer.lic

```diff
@@ -286,6 +308,68 @@ class LootProcess
     dispose_trash(checkright) if snap.last != checkright
   end
 
+  def check_wounds
```

Comment:
If most of this logic is shared with `healme`, I'd like to see you either leverage that script, or move this code to a shared location (a new `common-*` file?). What do you think?